build_debian.sh

```diff
@@ -411,6 +412,12 @@ sudo cp files/dhcp/snmpcommunity $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d
 sudo cp files/dhcp/vrf $FILESYSTEM_ROOT/etc/dhcp/dhclient-exit-hooks.d/
 sudo cp files/dhcp/dhclient.conf $FILESYSTEM_ROOT/etc/dhcp/
 
+## Config watchdog device and disable at startup
+sudo sed -i 's/#watchdog-device/watchdog-device/' $FILESYSTEM_ROOT/etc/watchdog.conf
+sudo sed -i 's/run_watchdog=1/run_watchdog=0/' $FILESYSTEM_ROOT/etc/default/watchdog
+sudo rm -rf $FILESYSTEM_ROOT/lib/systemd/system/wd_keepalive.service
+sudo rm -rf $FILESYSTEM_ROOT/etc/init.d/wd_keepalive
+
```

Comment:
- Enabling watchdog device in watchdog.conf and disabling watchdog by default.
- Watchdog can be enabled with sonic_platform API on need basis.
- Removed wd_keepalive support.